curator-client/pom.xml

```diff
@@ -36,7 +36,7 @@
 
     <properties>
         <osgi.import.package>
-            *
+            org.apache.zookeeper.*;version="[3.4,4.0)", *
```

Comment:
I have no background with OSGi. Could you explain this change? I mean, why "osgi.import.package" with '*' didn't do what you need and this change fixes it? Usually '*' is most general.